airflow/requirements.txt

```diff
@@ -1,13 +1,15 @@
-/tmp/airflow_provider_weaviate-1.0.0-py3-none-any.whl
+/tmp/airflow_provider_weaviate-0.0.1-py3-none-any.whl
```

Comment:
May I know why do we want to install the package this way? Should we push this to astronomer PyPI?

Reply:
These providers are not yet on pypi.  They will be replaced by llm-providers later.